exercises/knapsack/canonical-data.json

```diff
@@ -0,0 +1,116 @@
+{
+  "exercise": "knapsack",
+  "version": "1.0.0",
+  "comments": [
+    "Depending on the language, the input type can be modified"
+  ],
+  "cases": [
+    {
+      "description": "no items",
+      "property": "maximumValue",
+      "input": {
+        "maximumWeight": 100,
+        "items": {}
+      },
+      "expected": 0
+    },
+    {
+      "description": "one item, too heavy",
+      "property": "maximumValue",
+      "input": {
+        "maximumWeight": 10,
+        "items": [
+          { "weight": 100, "value": 1 }
+        ]
+      },
+      "expected": 0
+    },
+    {
+      "description": "five items (cannot be greedy by weight)",
+      "property": "maximumValue",
+      "input": {
+        "maximumWeight": 10,
+        "items": [
+          { "weight": 2, "value": 5 },
+          { "weight": 2, "value": 5 },
+          { "weight": 2, "value": 5 },
+          { "weight": 2, "value": 5 },
+          { "weight": 10, "value": 21 }
+        ]
+      },
+      "expected": 21
+    },
+    {
+      "description": "five items (cannot be greedy by price)",
```

Comment:
price or value? The latter seems more in-line with the rest of the exercise.

Reply:
Yes its value!